api/room/schema.py

```diff
@@ -183,7 +184,29 @@ def mutate(self, info, room_id, **kwargs):
         return DeleteRoom(room=exact_room)
 
 
+class UpdateFirebaseToken(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int(required=True)
+        firebase_token = graphene.String()
+    room = graphene.Field(Room)
+
+    def mutate(self, info, room_id, **kwargs):
+        MRM_URL = "https://www.google.com/"
```

Comment:
Good work @michael-basweti, kindly explain why you're using the google url as the `MRM_URL`

Reply:
that's just to show that after updating the token we can hit the mrm_push url and notify the service of the change. once the mrm push is done, we will have the url in the .env and and the above code will be edited so that the service can work. that's just an example

Reply:
@michael-basweti kindly read the URL from the `.env` file.

Reply:
@joshuaocero @philophilo is currently implementing that and I will include it once it's done

Reply:
@michael-basweti Has the URL been updated?